README.md

```diff
@@ -88,8 +91,9 @@ Output:
 }
 ```
 
-### Generating types
+<a name="generating-types"></a>
```

Comment:
Can you please explain what does this empty link do?

Reply:
Sorry for the late response! 

Since the headings text has changed, as did their auto-generated links, my solution was these "invisible" anchors to preserve the URL hash targets (visiting `./#generating-types` will auto-scroll down to the `<a name="generating-types">` tag).